be/src/vec/exec/join/vhash_join_node.cpp

```diff
@@ -180,9 +180,17 @@ Status HashJoinNode::init(const TPlanNode& tnode, RuntimeState* state) {
 #endif
 
     for (size_t i = 0; i < _runtime_filter_descs.size(); i++) {
-        RETURN_IF_ERROR(state->runtime_filter_mgr()->register_producer_filter(
-                _runtime_filter_descs[i], state->query_options(), &_runtime_filters[i],
-                _probe_expr_ctxs.size() == 1));
+        if (_runtime_filter_descs[i].has_remote_targets) {
+            RETURN_IF_ERROR(state->get_query_ctx()
```

Comment:
duplicate code, add a new method in runtimestate   register_producer_filter( _runtime_filter_descs[i], state->query_options(), &_runtime_filters[i],
                    _probe_expr_ctxs.size() == 1))

and then check if it is a local or global runtime filter inside the method.

Reply:
here is two different method，must in different mgr，must be check here